app/javascript/components/App.js

```diff
@@ -17,20 +17,33 @@ import MyUserProfile from './pages/MyUserProfile'
 import MyUserProfileEdit from './pages/MyUserProfileEdit'
 import UserProfile from './pages/UserProfile'
 
-
-
-
-
 class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      personalInfo: mockPersonalInfo
+    }
+  }
   render() {
     const {
       logged_in,
       sign_in_route,
       sign_out_route
     } = this.props
     return (
-      <>
-      <Header />
+      <Router>
+        <Header />
+        <Switch>
+          {/* Unprotected Routes */}
+          <Route
+            path="/userprofileindex"
+            render={ (props) => <UserProfile info={ this.state.personalInfo }/>}
+            >
+          </Route>
+
+
+          {/* Protected Routes */}
+        </Switch>
       <Home />
       <LearnMore />
       <DevTeam />
```

Comment:
Make sure you are consistent with your spacing inside the curly braces. Either have an extra space on either side or don't have one. No wrong answer, just be consistent.